CHANGELOG.md

```diff
@@ -2,11 +2,19 @@ From version 2.6.0, the sections in this file adhere to the [keep a changelog](h
 
 ## [Unreleased]
 
+## Version 2.15.7
+
```

Comment:
Changes must be added under [Unreleased], and then we create a new version number when we make a release.

```suggestion
```

Reply:
done